scripts/scr_purge_world/scr_purge_world.gml

```diff
@@ -21,16 +21,16 @@ function scr_purge_world(star, planet, action_type, action_score) {
 
 
 
-
+	// TODO - while I don't expect Surface to Orbit weapons retaliating against player's purge bombardment, it might still be worthwhile to consider possible situations
 
 	if (action_type=1){// Bombardment
-	    txt1="The heavens rumble and thunder as your ship";
+	    txt1=choose("Your cruiser and larger ship", "The heavens rumble and thunder as your ship");
 	    if (ships_selected>1) then txt1+="s";
-	    txt1+=" unload";
+	    txt1+=choose(" position themselves over the target in close orbit, and unleash", " unload");
 	    if (ships_selected=1) then txt1+="s";
-	     txt1+=" annihilation upon "+string(star.name)+" "+string(planet)+".  Even from space the explosions can be seen, clapping across the planet's surface.";
+		txt1+= $" annihilation upon {planet_numeral_name(planet, star)}. Even from space the explosions can be seen, {choose("tearing ground", "hammering", "battering", "thundering")} across the planet's surface.";
  
-	    if (star.p_large[planet]=0) then max_kill=action_score*15000000;// Population if normal
+	    if (star.p_large[planet]=0) then max_kill=action_score*15000000;// Population if normal, TODO as in scr_bomb_world, reconsider this formula
```

Comment:
One major difference with purge bombardment is that it never depopulates the planet entirely, even if all pop would be eliminated in a single bombardment otherwise.
Even old lava worlds, with 500-1500 population, still had people after several bombardments. It later got depopulated due to them all joining the PDF though.